best_practices/security.rst

```diff
@@ -1,7 +1,7 @@
 Security
 ========
 
-Authentication and Firewalls (i.e. Getting the User's Credentials)
+Authentication and Firewalls (e.g. Getting the User's Credentials)
```

Comment:
I think we need to be careful and not blindly replace all occurrences of "i.e." with "e.g." as they have different meanings. "e.g." is kind of equivalent to "for example" which is probably not meant here.